### PR TITLE
[15.0][MIG] mail_activity_done

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review :+1:

File: mail_activity_done/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This module implements the capability to keep activities that have been
+completed, for future reporting, by setting them with the boolean 'Done'.
+
+The activities that have been completed will not appear in the chatter.
```
Comment (emagdalenaC2i) [inline mail_activity_done/readme/DESCRIPTION.rst]: I think you can delete this line

**Review by emagdalenaC2i — APPROVED** [review]
LGTM, just some unimportant comments

**Comment by emagdalenaC2i** [comment]
Please, Could you add a short explanation in Usage.rst?

**Review by marcelsavegnago — APPROVED** [review]

**Comment by BernatPForgeFlow** [comment]
> Please, Could you add a short explanation in Usage.rst?

@emagdalenaC2i  Done! And the last line from DESCRIPTION.rst removed.

**Review by olgamarcocb — APPROVED** [review]
LGTM :+1:

**Review by etobella — APPROVED** [review]

File: mail_activity_done/LICENSE [hunk 2]
```diff
@@ -0,0 +1,859 @@
+
```
Comment (emagdalenaC2i) [inline mail_activity_done/LICENSE]: Please, delete this file

Comment (BernatPForgeFlow) [reply mail_activity_done/LICENSE]: Done

**Review by HaraldPanten — APPROVED** [review]
I'm sure you'll be able to merge this PR with your "coming soon" PSC powers, @etobella

**Pull request merged**